### ISSUE TITLE
letting omega team in gauntlet race spawn in any checkpoint

### ISSUE THREAD

**Issue by Lagetta** [blue-nebula/base#236]
Mode gauntlet is a mode where alpha team tries to make as many laps as possible while omega tries to delay them. After half of match time roles change.

The main issue is that omega is way underpowered against alpha.

Omega team always spawns at the start of the race while alpha on the last checkpoint. Meaning, alpha can just kill the omega camper and move on undisturbed, perhaps die a few times before killing them, which takes 1 sec to respawn. And what if the alpha racer is really skilled? You need to be even faster (not doing any mistakes, cuz omega spawns back) than that person to even stand a change before losing a point.

Basically Omega are in huge disadvantage.

After adding ability for omega to spawn everywhere, they can just suicide to quickly change positions, to counter alpha. Which I think is way better.
